src/test/java/roomescape/ReservationTest.java

```diff
@@ -0,0 +1,212 @@
+        Assertions.assertThat(response.jsonPath().getLong(ID)).isEqualTo(ONE);
+        this.theme = new Theme(response.jsonPath().getLong(ID), response.jsonPath().getString(NAME), response.jsonPath().getString(DESCRIPTION), response.jsonPath().getString(THUMBNAIL));
+    }
+
+    @Test
+    void 예약_관리_페이지를_랜더링한다() {
+
+        //when
+        ExtractableResponse<Response> response = RestAssured.given().log().all()
+                .when().get(RENDERING_RESERVATION_URL)
+                .then().log().all()
+                .extract();
+
+        //then
+        Assertions.assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+
+    }
+
+    @Test
+    void 예약을_등록한다() {
+
+        //given
+        Map<String, Object> reservation = new HashMap<>();
+        reservation.put(NAME, RESERVATION_NAME);
+        reservation.put(DATE, RESERVATION_DATE);
+        reservation.put(TIME_ID, ONE);
+        reservation.put(THEME_ID, ONE);
+
+
+        //when
+        ExtractableResponse<Response> response = RestAssured.given().log().all()
+                .contentType(ContentType.JSON)
+                .body(reservation)
+                .when().post(RESERVATIONS_API_URL)
+                .then().log().all()
+                .extract();
+
+        //then
+        Assertions.assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+        Assertions.assertThat(response.jsonPath().getLong(ID)).isEqualTo(LONG_ONE);
+    }
+
+    @Test
+    void 에약을_조회한다() {
+
+        //given
+        예약을_등록한다();
+
+        //when
+        ExtractableResponse<Response> response = RestAssured.given().log().all()
+                .when().get(RESERVATIONS_API_URL)
+                .then().log().all()
+                .extract();
+
+        Assertions.assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+        Assertions.assertThat(response.jsonPath().getList(WILD_CARD).size()).isEqualTo(ONE);
+    }
+
+    @Test
+    void 예약을_삭제한다() {
+
+        //given
+        예약을_등록한다();
+
+        //when
+        ExtractableResponse<Response> response = RestAssured.given().log().all()
+                .when().delete(RESERVATIONS_API_URL + PATH_VARIABLE_SUFFIX_URL)
+                .then().log().all()
+                .extract();
+
+        //then
+        Assertions.assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"가", "a", "가나다라마바사아자차카타파하거너더러머버서"})
+    void 예약_생성_중에_이름의_형식이_맞지_않는_경우_예외를_발생시킨다(String wrongNameExample) {
+
+        //given
+        ReservationRequest reservationRequest = new ReservationRequest(wrongNameExample, RESERVATION_DATE, this.time.getId(), theme.getId());
+
+        //when, then
+        Assertions.assertThatThrownBy(() -> new Reservation(null, reservationRequest.getName(), reservationRequest.getDate(), this.time, this.theme)).isInstanceOf(ReservationException.class).hasMessage(ErrorCode.INVALID_THEME_NAME_FORMAT_ERROR.getErrorMessage());
```

Comment:
```suggestion
        assertThatThrownBy(() -> new Reservation(
            null,
            reservationRequest.getName(),
            reservationRequest.getDate(),
            this.time,
            this.theme))
            .isInstanceOf(ReservationException.class)
            .hasMessage(ErrorCode.INVALID_THEME_NAME_FORMAT_ERROR.getErrorMessage());
```
줄 수가 많아지긴 하지만, 이렇게 줄바꿈을 해서 가독성을 높여보면 좋을 것 같아요.
테스트도 프로덕션 코드처럼 보기 편해야 이해가 잘되니까요.
`Assertions` 부분을 제거하는 것도 괜찮을 것 같구요.